Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -77,14 +92,20 @@ __Improved keyboard navigation__
 - When a `T:System.Windows.Forms.ComboBox` control has `P:System.Windows.Forms.ComboBox.DropDownStyle` set to `F:System.Windows.Forms.DropDownStyle.DropDownList` and is the first control in the tab order on the form, it now displays a focus rectangle when the parent form is opened using the keyboard. Before this change, keyboard focus was on this control, but a focus indicator was not rendered.
 
 __Improved Narrator support__
-- The `T:Sysetm.Windows.Forms.MonthCalendar`control has added support for assistive technologies to access the control, inclding the ability for Narrator to read the value of the control when previously it could not.
+- The `T:System.Windows.Forms.MonthCalendar` control has added support for assistive technologies to access the control, including the ability for Narrator to read the value of the control when previously it could not.
+- The `T:System.Windows.Forms.CheckedListBox` control now notifies Narrator when the `P:System.Windows.Forms.CheckState` property has been changed. Previously, Narrator did not recieve notification and as a result users would not be informed that the `P:System.Windows.Forms.CheckState` had been updated.
+- The `T:System.Windows.Forms.LinkLabel` control has changed the way it notifies Narrator of the text of in the control. Previously, Narrator announced this text twice and read "&" symbols as real text even though they are not visible to a user. The duplicated text was removed from the Narrator announcements, as well as unnecessary & symbols.
```

Comment:
ampersand should be either in quotes in both places or with no quotes in both places :)